commit 884c0aed054302cb1cbb71c2b21c12df92a7e6e4
Author: Karel Klic <kklic@redhat.com>
Date:   Thu Apr 21 08:59:38 2011 +0200

    Do not allow to submit backtraces with rating 0 (or some other unexpected rating).

diff --git a/src/gui-wizard-gtk/wizard.c b/src/gui-wizard-gtk/wizard.c
index 44dc11b..0a57f04 100644
--- a/src/gui-wizard-gtk/wizard.c
+++ b/src/gui-wizard-gtk/wizard.c
@@ -1005,8 +1005,7 @@ static void check_backtrace_and_allow_send(void) //TODO: rename, this checks rat
                 add_warning(_("The backtrace is incomplete, please make sure you provide the steps to reproduce."));
                 warn = true;
                 break;
-            case '2':
-            case '1':
+            default:
                 //FIXME: see CreporterAssistant: 394 for ideas
                 add_warning(_("Reporting disabled because the backtrace is unusable."));
                 send = false;
